.gitignore

```diff
@@ -39,6 +39,8 @@ tests/libcyclus-orig.h5
 tests/libcyclus-orig.sqlite
 tests/libcyclus-test.h5
 tests/libcyclus-test.sqlite
+test/dummy.json
```

Comment:
I think this should be plural: "tests/dummy.json" and "tests/dummy.h5" . (There is no "test" directory it's "tests".)